src/SQLProvider.Runtime/Providers.MsSqlServer.Ssdt.fs

```diff
@@ -165,32 +165,31 @@ module MSSqlServerSsdt =
     let tryFindMapping (dataType: string) =
         typeMappingsByName.TryFind (dataType.ToUpper())
 
-    let tryFindMappingOrVariant (dataType: string) =
-        match typeMappingsByName.TryFind (dataType.ToUpper()) with
+    let rec tryFindMappingOrVariant (uddts: SsdtUserDefinedDataType) (dataType: string) =
+        let dataType = dataType.ToUpper()
+        match typeMappingsByName.TryFind dataType with
         | Some tm -> tm
-        | None -> (typeMappingsByName.TryFind "SQL_VARIANT").Value
```

Comment:
While you're in there, can you improve this line so that it doesn't call `TryFind` and `Value` (I was probably rushing):

```F#
| None -> typeMappingsByName["SQL_VARIANT"]
```

Reply:
Sure thing, the map should always have it at this point, so makes sense.

Changed it on this commit:
https://github.com/fsprojects/SQLProvider/pull/779/commits/f0592016fed253502e93ec6802b2e431a17dd47e